src/main/js/helpers/axis.js

```diff
@@ -329,35 +428,41 @@ const getAverageTicksCount = (rangeY, rangeY2) => {
  * @param {number} lowerLimit - Lower limit of the Y or Y2 Axis
  * @param {number} upperLimit - Upper limit of the Y or Y2 Axis
  * @param {number} ticksCount - Number of ticks between the upper and lower limits
+ * @param {boolean} allowCalibration - Whether this property is true or false
  * @returns {(Array)} returns array of values to be used as tick labels
  */
 const generateYAxesTickValues = (
     lowerLimit,
     upperLimit,
-    ticksCount = constants.DEFAULT_TICKSCOUNT
+    ticksCount = constants.DEFAULT_TICKSCOUNT,
+    allowCalibration = true
 ) => {
     ticksCount = Math.abs(ticksCount);
     const tickValues = [];
 
     // use the d3,js nice function to round off the upper and lower limits
     // to multiples of 2, 5 or 10
-    const [newLowerLimit, newUpperLimit] = d3
-        .scaleLinear()
-        .domain([lowerLimit, upperLimit])
-        .nice()
-        .domain();
+    // let [newLowerLimit, newUpperLimit] = [lowerLimit, upperLimit];
```

Comment:
is this line necessary in here as comment?

Reply:
thanks for pointing that out, removed here:
https://github.com/cerner/carbon-graphs/pull/249/commits/3b13c155f716f18e013f983f062069f8baaa8cd2